config/vufind/config.ini

```diff
@@ -528,15 +528,17 @@ encrypt_ils_password = false
 
 ; This is the key used to encrypt and decrypt catalog passwords.  This must be
 ; filled in with a random string value when encrypt_ils_passwords is set to true.
+; aes (default) requires a key exactly 32 characters long.
+; 32 chars ->        "--------------------------------"
 ils_encryption_key = false
 
 ; This is the algorithm used to encrypt and decrypt catalog passwords.
 ; A symmetrical encryption algorithm must be used.
 ; You can use openssl_get_cipher_methods() to see available options on your system.
-; Common choices: blowfish (default), aes
+; Common choices: aes (default), cast, sm4
 ; If you want to convert from one algorithm to another, run this from $VUFIND_HOME:
 ;   php public/index.php util switch_db_hash oldhash:oldkey (or none) newhash:newkey
-;ils_encryption_algo = "blowfish"
+;ils_encryption_algo = "aes"
```

Comment:
As noted earlier, I think we need to keep blowfish as the default fallback for the VuFind\Db\Row\User class, since otherwise we'll misinterpret legacy configuration files and try to apply the wrong algorithm. Instead, maybe what we need to do is uncomment the aes line by default in the configuration file, to enforce a reasonable default for new installations. Maybe we can change (default) in the comment to (recommended).

Reply:
That's a good point.

Reply:
Although it is disabled by default... wouldn't we want to encourage good practices for people who haven't messed with the default (disabled) config?

Reply:
My thought is that at some point, we should throw an exception if the algorithm is undefined. So maybe in VuFind 9.0, we issue a deprecation warning, like "ils_encryption_algo undefined; defaulting to blowfish" and then in VuFind 10.0, it's a fatal error to turn on encryption without specifying an algorithm.

Reply:
Do we want the deprecation warning in this PR?

Reply:
@crhallberg, I think this detail still needs to be done. Can you double-check?

Reply:
Forgot to push a change but I'm completely confident this is the best way to display a warning.